sql/chunk.sql

```diff
@@ -67,3 +67,10 @@ RETURNS TABLE(chunk_id INTEGER, hypertable_id INTEGER, att_num INTEGER, nullfrac
 slot1numbers FLOAT4[], slot2numbers FLOAT4[], slot3numbers FLOAT4[], slot4numbers FLOAT4[], slot5numbers FLOAT4[],
 slotvaluetypetrings CSTRING[], slot1values CSTRING[], slot2values CSTRING[], slot3values CSTRING[], slot4values CSTRING[], slot5values CSTRING[])
 AS '@MODULE_PATHNAME@', 'ts_chunk_get_colstats' LANGUAGE C VOLATILE;
+
+CREATE OR REPLACE FUNCTION _timescaledb_internal.create_chunk_table(
+       hypertable REGCLASS,
+       slices JSONB,
+       schema_name NAME,
+       table_name NAME)
+RETURNS OID AS '@MODULE_PATHNAME@', 'ts_chunk_create_empty_table' LANGUAGE C VOLATILE STRICT;
```

Comment:
When this function is called remotely from the AN, then the OID can't be used. Just pointing this out since it is easy to make the mistake if this is the signature. Other functions, like `create_chunk` elided the return of an OID in favor of schema+name for this reason.

Reply:
Also wondering if the return type should actually be `REGCLASS` (if we stick with OIDs)?

Reply:
Good point about not returning `OID` or `REGCLASS` for remotely called function.
Will it be suitable to return `BOOL`?